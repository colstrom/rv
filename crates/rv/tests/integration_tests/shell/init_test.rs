use crate::common::RvTest;
use insta::assert_snapshot;

#[test]
fn test_shell_init_succeeds() {
    let test = RvTest::new();
    let output = test.rv(&["shell", "init"]);

    assert_snapshot!(output.normalized_stdout());
    assert!(output.success());
}
