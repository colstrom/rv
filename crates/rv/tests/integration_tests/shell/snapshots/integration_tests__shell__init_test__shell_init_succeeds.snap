---
source: crates/rv/tests/integration_tests/shell/init_test.rs
expression: output.normalized_stdout()
---
autoload -U add-zsh-hook
_rv_autoload_hook () {
    eval "$(/tmp/bin/rv shell env)"
}
add-zsh-hook chpwd _rv_autoload_hook
_rv_autoload_hook
